pkg/prometheus/operator.go

```diff
@@ -182,10 +197,6 @@ func (c *Operator) Run(stopc <-chan struct{}) error {
 	go c.cmapInf.Run(stopc)
 	go c.psetInf.Run(stopc)
 
-	for !c.promInf.HasSynced() || !c.smonInf.HasSynced() || !c.cmapInf.HasSynced() || !c.psetInf.HasSynced() {
```

Comment:
FYI I removed this as it didn't seem to do anything other than possibly infinite loop if ^C before initial syncs had run.